openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -80,26 +80,6 @@ class CoursewareInformation(RetrieveAPIView):
 
     serializer_class = CourseInfoSerializer
 
-    def _check_access(self, user, overview, is_staff):
```

Comment:
This was a function we added to a previous PR that we're now folding into `check_course_access`